models/games.model.js

```diff
@@ -6,7 +6,14 @@ exports.selectCategories = () => {
 
 exports.selectReviewById = (reviewId) => {
   return db
-    .query(`SELECT * FROM reviews WHERE review_id = $1;`, [reviewId])
+    .query(
+      `SELECT reviews.review_id, title, review_body, designer, review_img_url, reviews.votes, category, owner, reviews.created_at, 
+      COUNT(comment_id) AS comment_count FROM reviews 
+      JOIN comments ON comments.review_id = reviews.review_id
```

Comment:
Maybe using LEFT JOIN instead of JOIN in case some of the reviews have no comments